newton/_src/viewer/viewer.py

```
@@ -334,6 +340,63 @@ def log_contacts(self, contacts, state):
         self.log_lines("/contacts", starts, ends, colors)
 
+    def log_isosurface(self, isosurface_data, penetrating_only: bool = True):
+        """
+        Render the isosurface triangles as wireframe lines from SDFHydroelastic collision detection.
+
+        Args:
+            isosurface_data: An IsosurfaceData instance containing vertex arrays for visualization,
+                or None if hydroelastic collision is not enabled.
+            penetrating_only: If True, only render penetrating contacts (depth > 0).
+        """
+        from .kernels import compute_isosurface_lines  # noqa: PLC0415
```

Ruff (0.14.8)

Unused `noqa` directive (non-enabled: `PLC0415`)

Remove unused `noqa` directive

(RUF100)